stroyprombeton/tests/tests_models.py

```diff
@@ -1,16 +1,37 @@
 from django.test import TestCase, tag
 
+from pages import models as pages_models
 from stroyprombeton import models as stb_models
 
 
 @tag('fast')
-class Category_(TestCase):
+class Option_(TestCase):
 
     fixtures = ['dump.json']
```

Comment:
It seems you should remove this, because you create custom data for this test case